etc/init.d/zfs-mount.in

```diff
@@ -57,7 +57,7 @@ do_depend()
 	# bootmisc will log to /var which may be a different zfs than root.
 	before bootmisc logger
 
-	after procfs zfs-import sysfs procps
```

Comment:
Why was `procfs` removed? if `/etc/mtab` is a symlink to `/proc/mounts` or `/etc/mtab` is not there, `libzfs_mnttab_find()` will check `/proc/mounts` to see if things are already mounted before sending the request to the kernel. This should be used by `zfs mount -a`.